telemetry/specs/INT.mdk

```diff
@@ -910,6 +915,170 @@ from (shim header length \* 4).
 For INT over Geneve it is 8 bytes subtracted from (length in Geneve tunnel
 option header \* 4).
 
+
+INT Metadata Header and Metadata Stack (Version = 2):
```

Comment:
@mhira1 I was meaning to propose make a 1.x version cut before merging these changes, and move INT.mdk to version2 draft. Then we don't need to list both v1 and v2 in the same doc.

Reply:
Makes sense. Let's remove this second paragraph on v2 header then.